wd_demo.au3

```diff
@@ -1139,27 +1139,34 @@ EndFunc   ;==>UserTesting
 
 Func UserFile()
 	; Modify the contents of UserTesting.au3 (or create new one) to change the code being executed.
-	; Changes can be made and executed without restarting this script
-	Local $sScriptFileFullPath = FileOpenDialog('Choose testing script', @ScriptDir, 'AutoIt script file (*.au3)', $FD_FILEMUSTEXIST, 'UserTesting.au3')
-	If @error Then Return SetError(@error, @extended)
+	; Changes can be made and executed without restarting this script,
+	;	you can even repeat "UserFile" processing without closing browser
+	Local Const $aEmpty1D[UBound($_VAR)] = []
+	Local $sScriptFileFullPath, $aCmds
 
-	Local $aCmds = FileReadToArray($sScriptFileFullPath)
-	If @error Then Return SetError(@error, @extended)
+	Do
+		$_VAR = $aEmpty1D ; clean up the globally declared $_VAR to ensure repeatable test conditions
+		$sScriptFileFullPath = FileOpenDialog('Choose testing script', @ScriptDir, 'AutoIt script file (*.au3)', $FD_FILEMUSTEXIST, 'UserTesting.au3')
+		If @error Then Return SetError(@error, @extended)
 
-	Local Const $aEmpty1D[UBound($_VAR)] = []
-	$_VAR = $aEmpty1D ; clean up the globally declared $_VAR to ensure repeatable test conditions
+		$aCmds = FileReadToArray($sScriptFileFullPath)
+		If @error Then Return SetError(@error, @extended)
+
+		For $sCmd In $aCmds
+			; Strip comments
+			; https://www.autoitscript.com/forum/topic/157255-regular-expression-challenge-for-stripping-single-comments/?do=findComment&comment=1138896
+			$sCmd = StringRegExpReplace($sCmd, '(?m)^(?:[^;"'']|''[^'']*''|"[^"]*")*\K;.*', "")
+			If $sCmd Then Execute($sCmd)
+		Next
+
+	Until $IDNO = MsgBox($MB_YESNO + $MB_TOPMOST + $MB_ICONQUESTION + $MB_DEFBUTTON2, 'Question', 'Do you want to repeat "UserFile" processing ?')
```

Comment:
Perhaps change the question to "Do you want to process another file?"

Reply:
As you suggested [question changed](https://github.com/Danp2/au3WebDriver/pull/464/commits/86f9d4e4dcd6e9e95f8cd7839dd4184244727edb)